src/main/java/mate.academy/Main.java

```diff
@@ -4,13 +4,15 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
```

Comment:
The import statement for 'ProductServiceImpl' is unnecessary because you should rely on interfaces, not concrete implementations. The injector should be able to determine the implementation to use without a direct reference in the 'Main' class.